src/style.css

```diff
@@ -2,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src:
+    url(../src/fonts/Avenir-Book.ttf) format('truetype'),
+    url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.body {
```

Comment:
The class name `.body` can be misleading as it suggests styling for the HTML `body` element. Consider using a different class name that better represents the content or purpose of the elements it styles.